src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx

```diff
@@ -142,32 +142,15 @@ export class BiggiveCampaignCardFilterGrid {
     this.doSearchAndFilterUpdate.emit(this.getSearchAndFilterObject());
   };
 
-  private getSearchAndFilterObject(): {
-    searchText: string;
-    sortBy: string;
-    filterCategory: string;
-    filterBeneficiary: string;
-    filterLocation: string;
-    filterFunding: string;
-  } {
-    const event: {
-      searchText: string | null;
-      sortBy: string | null;
-      filterCategory: string | null;
-      filterBeneficiary: string | null;
-      filterLocation: string | null;
-      filterFunding: string | null;
-    } = {
+  private getSearchAndFilterObject() {
```

Comment:
no need for an explicit return type here I think, since it's easy to infer both for the compiler and human readers.